terraform/ecs_ec2/daemon/main.tf

```diff
@@ -11,7 +11,7 @@ module "basic_components" {
 }
 
 resource "aws_ecs_cluster" "cluster" {
-  name = "cwagent-integ-test-cluster-${module.common.testing_id}"
+  name = "${local.basename}-${module.common.testing_id}"
```

Comment:
I have no issue adding more tags.
Just curious if we could have just searched for existing `ClusterName` tags containing (or starting with)  `cwagent-integ-test-cluster`?